### DISCUSSION TITLE
Use phospor in NextJS

### DISCUSSION THREAD

**Discussion by brianrangeldev** [phosphor-icons/react#53] [General]
how to use icons in nextjs project, I've tried the normal way but it doesn't work. ?

**Comment by rektdeckard** [comment]
For the time being, it can only be used in Client components. We depend on react `Context`, and I don't have a good way of injecting "use client" directives into the bundled library artifacts. I have a PR open #47 which aims to address this without breaking the context stuff for existing users, but it needs some testing.

**Comment by rektdeckard** [comment]
An experimental build is now live on NPM under version `2.1.3`, and also installable under the `@next` tag. Check out #64 for details on usage. We plan to mainstream this soon, once we've had time to get any feedback.

**Comment by rektdeckard** [comment]
Closing as duplicate of #46
